### PR TITLE
20.11.0 changelog

### PR DESCRIPTION
**Pull request by jframe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 
-## Next Release
+## 20.11.0
 
 ### Features Added
 
 - Interlock/Armory II HSM keystore support
-- Eth2 slashing protection data able to be imported from json file (Interchange format V5)
+- Eth2 slashing protection data able to be exported and imported from json file (Interchange format V5)
+- Eth2 signing API can now return response in json format
```
Comment (rain-on) [inline CHANGELOG.md:9]: ... returned body matches incoming request content-type (json or text) [or something?]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+- Genesis validators root validation for eth2 signing requests. The GVR is set on first eth2 signing request.
```
Comment (rain-on) [inline CHANGELOG.md:10]: Only able to sign for a single Genesis validators root (which is defined by the first received request after creation)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,20 @@
...
+- Use a low watermark in Eth2 slashing protection and import/export
+
```
Comment (rain-on) [inline CHANGELOG.md:12]: * Do not sign below watermark (regardless of if matching existing entry)
* Resolved help text anomalies

Comment (jframe) [reply CHANGELOG.md:12]: done

File: CHANGELOG.md (hunk 2 above)
Comment (jframe) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (jframe) [reply CHANGELOG.md]: done

**Review by rain-on — APPROVED** [review]

**Pull request merged**